support.html

```diff
@@ -0,0 +1,54 @@
+<!--#include virtual="includes/_header.htm" -->
+<body class="page-contact ">
+<!--#include virtual="includes/_top.htm" -->
+<div class="content">
+	<!--#include virtual="includes/_nav.htm" -->
+	<div class="right">
+		<h1 class="content-title">Support</h1>
+		<h3>Community support</h3>
+
+		<p>
+			This is an open source project so the amount of time community members have available to help resolve your issue
+			is often limited as help is provided on a volunteer basis from
+			<a href="https://www.apache.org/foundation/how-it-works/#hats" target="_blank">individuals</a>.
+			However, it is free, and you are often able to discuss issues with the developers who actually wrote the code
+			you are running.
+		</p>
+		<p>
+			If you want community support then please <a href="https://kafka.apache.org/contact">contact us</a>.
```

Comment:
It might be good to be a bit more specific here. For example most support related questions should probably go to the `users@apache.kafka.org` mailing list specifically

Reply:
Another common source of help is the Confluent Community Slack. Yes, it's run by Confluent, but it's free and is fairly active with people, both users and devs, from across a wide range of companies. Not sure if we'd need to get permission to include that or not.

I also see people ask and get good answers on StackOverflow, though generally speaking the users mailing list and Confluent Community Slack are the most active in my experience

Reply:
There is an existing ASF Slack and a Kafka channel so why not using it?

Reply:
I will mention and add a link to StackOverFlow.

Reply:
I know @ableegoldman is trying to list all helpful resources here, but I don't think adding `Confluent Community Slack` or `StackOverflow` is appropriate here. `Confluent Community Slack` is run by Confluent, so it is not a neutral recourse. About the `StackOverflow`, I think if we list `StackOverflow` here, does that imply `users@apache.kafka.org` or `dev@apache.kafka.org` are not useful? I think, as a community, we should recommend users to ask/answer questions in the community, instead of in the 3rd party site.